randovania/generator/generator.py

```diff
@@ -148,7 +152,16 @@ async def _create_pools_and_fill(
                 status_update,
             )
             _validate_pickup_pool_size(new_pool.pickups, new_pool.game, new_pool.configuration)
+
+            # All majors required
+            if player_preset.configuration.logical_pickup_placement is not LogicalPickupPlacementConfiguration.MINIMAL:
```

Comment:
I think I like better if it were something like this:
```python
game.victory_condition = force_logical_placement(new_pool.pickups, new_pool.game.resource_database, player_preset.configuration.logical_pickup_placement)
```

Reply:
in that case the function should also be renamed IMO.

Reply:
to what?

Reply:
`get_victory_condition_with_additional_pickups`?
`add_pickups_to_victory_condition`?

Probably something like that would work

Reply:
I was thinking more in terms of `victory_condition_for_pickup_placenent` but am also fine with something else.
Name should make clear that youre returning a new victory condition (so no `add` in Henrique's example).